parsers/CA_QC.py

```diff
@@ -25,8 +25,10 @@ def if_exists(elem: dict, etype: str):
             "thermique": "thermal",
             "solaire": "solar",
             "eolien": "wind",
-            "autres": "unknown",
+            # autres is all renewable, and mostly biomass.  See Github    #3218
+            "autres": "biomass",
             "valeurs": "values",
+            "biomass":0.0
```

Comment:
```suggestion
```

This isn't needed and is a bit confusing given assignment of "autres" to "biomass" above.

Reply:
python test_parser.py CA-QC is passing, so I will push my commits.